### ISSUE TITLE
Add example of lithium plating for composite electrodes

### ISSUE THREAD

**Issue by brosaplanella** [pybamm-team/PyBaMM#4273]
When writing the release notes I realised there is no example of plating for composite electrodes. This is tricky, especially as additional parameters are required. Having such an example would be very useful.

@DrSOKane, if you have the time to write it that would be great, but otherwise could you post here a source for the parameter values in case someone wants to pick up this issue?

**Comment by mohammedasher** [comment]
@brosaplanella @DrSOKane

I noticed that while the lithium plating model for composite electrodes is implemented, the `chen2020_composite` parameter set does not currently include the plating parameters. Would it be helpful to copy the plating parameters from `OKane2022 ` to `chen2020_composite ` and use the same parameter values for both the primary and secondary phases?

If that sounds reasonable, I could also write a small example notebook demonstrating degradation (SEI and plating) in composite electrodes. Let me know your thoughts!

**Comment by brosaplanella** [comment]
Yes, that sounds good to me. Include a comment in the file stating that the parameters are pulled from another paper (i.e. not provided in the Chen2020 paper).

**Comment by DrSOKane** [comment]
Hi Ferran/Asher, I remember looking into this a while back and finding it very difficult to find papers where this had been done! I did find one, but as usual some parameters were missing.
